src/Magnum/DartIntegration/Test/DartSkeletonTest.cpp

```diff
@@ -0,0 +1,267 @@
+#include <dart/dynamics/BallJoint.hpp>
+#include <dart/dynamics/BodyNode.hpp>
+#include <dart/dynamics/BoxShape.hpp>
+#include <dart/dynamics/CylinderShape.hpp>
+#include <dart/dynamics/EllipsoidShape.hpp>
+#include <dart/dynamics/RevoluteJoint.hpp>
+#include <dart/dynamics/Skeleton.hpp>
+#include <dart/simulation/World.hpp>
+#include <dart/utils/urdf/urdf.hpp>
+
+#include <Corrade/TestSuite/Tester.h>
+
+#include <Magnum/SceneGraph/MatrixTransformation3D.h>
+#include <Magnum/SceneGraph/Object.hpp>
+#include <Magnum/SceneGraph/SceneGraph.h>
+#include <Magnum/Trade/ImageData.h>
+#include <Magnum/Trade/MeshData3D.h>
+#include <Magnum/Trade/PhongMaterialData.h>
+#include <Magnum/Trade/TextureData.h>
+
+#include <Magnum/DartIntegration/ConvertShapeNode.h>
+#include <Magnum/DartIntegration/DartSkeleton.h>
+
+#include <Magnum/DartIntegration/Test/configure.h>
+
+const double default_height = 1.0; // m
```

Comment:
Coding style: since these are constants, can they be named CamelCase and `constexpr`? So e.g.

```cpp
constexpr Double DefaultHeight = 1.0;
```

Thank you :)